src/SFA.DAS.ApplyService.Web/Views/Application/Pages/Index.cshtml

```diff
@@ -216,22 +218,41 @@
 
                         @if (!Model.HideCTA)
                         {
-                            <button type="submit" class="govuk-button">@Model.CTAButtonText</button>
+                            if (Model.PageOfAnswers != null && Model.PageOfAnswers.Count > 0 && Model.PageOfAnswers.First().Answers.Count(x => x.Value.ToLower().Contains(".pdf"))>0)
```

Comment:
As we discussed this is being restructured so it checks for type of file upload

Reply:
Will do in the story immediately following this APR-721